src/pykeen/models/base.py

```diff
@@ -170,6 +170,10 @@ def _reset_parameters_(self):  # noqa: D401
         """Reset all parameters of the model in-place."""
         raise NotImplementedError
 
+    @abstractmethod
```

Comment:
Do we want to enforce that each model implements this method, even if most models may not enforce any constraints?

Reply:
You mean leave as an empty function instead of a proper abstract function? Or have a check inside the model to see if it's been implemented?

Reply:
I see what you mean. Addressed in 5a3ff13d